apps/src/studio/blocks.js

```diff
@@ -828,6 +829,76 @@ exports.install = function (blockly, blockInstallOptions) {
       yParam + ');\n';
   };
 
+  blockly.Blocks.studio_addGoal = {
+    // Block for adding a goal flag at a specified position
+    helpUrl: '',
+    init: function () {
+      var dropdown;
+      if (allowSpritesOutsidePlayspace) {
+        dropdown = new blockly.FieldDropdown(this.VALUES_EXTENDED);
+        dropdown.setValue(this.VALUES_EXTENDED[4][1]); // default to top-left
+      } else {
+        dropdown = new blockly.FieldDropdown(this.VALUES);
+        dropdown.setValue(this.VALUES[1][1]); // default to top-left
+      }
+      this.setHSV(184, 1.00, 0.74);
+      this.appendDummyInput()
+        .appendTitle(msg.addGoal());
```

Comment:
Ah, good point. I was just blindly mirroring the set sprite position logic. I'll remove `VALUES_EXTENDED`.

Do you have an example of someplace we're using that workaround?